homeworks/vulmanjanos/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,38 @@
+from pprint import pprint
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+program_languages = input("Mike, please give me 4 different program languages, seprated by a comma!")
+print(program_languages)
+program_language_list = program_languages.split(",") # listává konvertálás és elválasztás vesszővel
+user_info.update({"skills": program_language_list}) # lista hozzáadása user infóhoz
+user_info["favourite_meals"].sort() #lista abc sorrendbe rakása
+pprint(user_info)
+pprint(user_info["favourite_meals"][-2]) #az updatelt listáról az utolsó előtti elem kiírása
+user_info["favourite_meals"].append("spagetthi") # hozzáadom a listához a spagetthit
+pprint(user_info["favourite_meals"])
+user_info["favourite_meals"].extend(["pizza", "sushi"]) # hozzáadom újra a pizzát és a sushit
+pprint(user_info["favourite_meals"])
+user_info["favourite_meals"].remove("pizza") # leszedem a pizzát
```

Comment:
itt is az értékekre hivatkozol, a felcseréléshez próbáld meg ezt:
```
user_info["favourite_meals"][0], user_info["favourite_meals"][-1] = (
    user_info["favourite_meals"][-1],
    user_info["favourite_meals"][0],
)
```

a megoldás itt is az indexek használatában rejlik, és hogy 1 soron egyszerre változtatjuk a 0. elem értékét a -1.-re és fordítva.